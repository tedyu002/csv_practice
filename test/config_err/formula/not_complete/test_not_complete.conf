CSV_INPUT=test.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
HEADERS=H1 INTEGER, H2 BOOL, H3 DOUBLE, H4 CHAR(4), H5 VARCHAR(100), H6 DATETIME
CSV_FORMULA=Print H8[0] +
SORT_HEADERS=H6,H5,H4,H3,H2,H1
SORT_ORDER=ASC
