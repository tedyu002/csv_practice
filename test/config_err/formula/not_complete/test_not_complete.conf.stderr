Failed to parse CSV_FORMULA 'Print H8[0] +'.
config 'test_not_complete.conf' parse error
